src/jit/importer.cpp

```diff
@@ -2903,14 +2915,12 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
     //
     // We disable the inlining of instrinsics for MinOpts.
     //
-    if  (opts.compDbgCode || opts.MinOpts())
+    if  ((opts.compDbgCode || opts.MinOpts()) && !(*pMustExpand))
```

Comment:
I would suggest to abstract this policy in a routine e.g. IsIntrinsicExpansionDisabled(CorInfoIntrinsics intrinsicID)